src/scripts.js

```diff
@@ -98,43 +124,13 @@ function checkFavoriteRecipes() {
     hide(noFavoritesMessage);
     show(favoriteRecipesContainer);
     renderRecipeCards(favoriteRecipesContainer, user.favoriteRecipes);
+
   } else {
-    show(noFavoritesMessage);
     hide(favoriteRecipesContainer);
+    show(noFavoritesMessage);
   }
 }
 
-function getData() {
```

Comment:
I think this getData function is not at the top. It might have gotten lost in the copy pasting at the top.